database/migration/src/main/resources/changesets/migration/workflow_store/replace_empty_custom_labels.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE WORKFLOW_STORE_ENTRY
+SET CUSTOM_LABELS = "{}"
+WHERE CUSTOM_LABELS = "";
```

Comment:
When turning on the 25_hotfix against a 24 database, if I stop cromwell after the liquibase migrations finish and look at the workflow store table, if I run a query select * from workflow_store_entry where set custom_labels=NULL, I get no matches. If I try to match for empty strings "", then I can see every workflow that was running prior to the migration.